app/lib/actions/pulp/repository/sync.rb

```diff
@@ -30,8 +31,6 @@ def invoke_external_task
             sync_options[:feed] = input[:source_url] if input[:source_url]
             sync_options[:validate] = !(SETTINGS[:katello][:pulp][:skip_checksum_validation])
 
-            sync_options.merge(input[:options]) if input[:options]
```

Comment:
part of the reason for the 'merge' here was to have options coming in override anything set via SETTINGS, basically overriding the defaults.  That said, this should have been merge!.  I'm curious though how both I and the reviewer got this to work originally?